src/main.rs

```diff
@@ -611,6 +619,24 @@ fn parse_opts_into_struct(
         };
     }
 
+    if let Some(keep) = matches.get_one::<String>("keep") {
+        let mut keep_display = false;
+        let mut names = keep
+            .split(',')
+            .filter_map(|name| {
+                if name == "display" {
+                    keep_display = true;
+                    return None;
+                }
+                Some(parse_chunk_name(name))
+            })
+            .collect::<Result<IndexSet<_>, _>>()?;
+        if keep_display {
+            names.extend(StripChunks::KEEP_SAFE.iter().cloned());
```

Comment:
It may be a good idea to standardize the terms used throughout the code and rename this and similar identifiers to `KEEP_DISPLAY`, as it can make things easier to follow. What do you think about this?

Reply:
Ah, I did mean to do that. Must have forgot. Changed now :)